pkg/template/config_context_test.go

```diff
@@ -425,6 +451,12 @@ func Test_localImageName(t *testing.T) {
 			image:    "redis:latest",
 			expected: "redis:latest",
 		},
+		{
+			name:     "do not rewrite public image",
```

Comment:
this test name seems opposite of the behavior

```suggestion
			name:     "rewrite public image when ProxyPublicImages: true",
```

Reply:
thanks.  copy/paste.